homeassistant/components/synology_dsm/__init__.py

```diff
@@ -214,32 +211,82 @@ def _async_migrator(entity_entry: entity_registry.RegistryEntry):
             entry, data={**entry.data, CONF_MAC: network.macs}
         )
 
-    # setup DataUpdateCoordinator
-    async def async_coordinator_update_data_surveillance_station():
-        """Fetch all surveillance station data from api."""
+    async def async_coordinator_update_data_cameras():
+        """Fetch all camera data from api."""
+        if not hass.data[DOMAIN][entry.unique_id][SYSTEM_LOADED]:
+            raise UpdateFailed("System not fully loaded")
+
+        if SynoSurveillanceStation.CAMERA_API_KEY not in api.dsm.apis:
+            return None
+
         surveillance_station = api.surveillance_station
+
         try:
             async with async_timeout.timeout(10):
                 await hass.async_add_executor_job(surveillance_station.update)
         except SynologyDSMAPIErrorException as err:
+            _LOGGER.debug(
+                "async_coordinator_update_data_cameras() - exception: %s", err
```

Comment:
This logging message format is non conventional. We start logging messages with capital letter and rarely include the name of methods in the message. Make a short statement what happened instead.